stable/datadog/values.yaml

```diff
@@ -424,6 +424,11 @@ clusterAgent:
   #
   # readinessProbe:
 
+  ## @param datadog-cluster.yaml - object - optional
+  ## Specify custom contents for the datadog cluster agent config (datadog-cluster.yaml).
+  #
+  # datadog-cluster.yaml: {}
```

Comment:
Took it for a spin, I think this should be:
`datadog_cluster_yaml`

I used `datadog_cluster_yaml: {"api_key":"foo"}` and this resulted in:

```
==> v1/ConfigMap
NAME                                     DATA  AGE
datadog-monitoring-cluster-agent-config  1     2s
```
which contained:
```
$ kubectl get cm datadog-monitoring-cluster-agent-config -oyaml
apiVersion: v1
data:
  api_key: foo
kind: ConfigMap
[...]
```